docs/archives/CHANGELOG_24.02-to-24.12.md

```diff
@@ -1,5 +1,190 @@
 # Change log
-Generated on 2025-02-17
+Generated on 2025-04-10
+
+## Release 24.12
+
+### Features
+|||
+|:---|:---|
+|[#11630](https://github.com/NVIDIA/spark-rapids/issues/11630)|[FEA] enable from_json and json scan by default|
+|[#11709](https://github.com/NVIDIA/spark-rapids/issues/11709)|[FEA] Add support for `MonthsBetween`|
+|[#11666](https://github.com/NVIDIA/spark-rapids/issues/11666)|[FEA] support task limit profiling for specified stages|
+|[#11662](https://github.com/NVIDIA/spark-rapids/issues/11662)|[FEA] Support Apache Spark 3.4.4|
+|[#11657](https://github.com/NVIDIA/spark-rapids/issues/11657)|[FEA] Support format 'yyyyMMdd HH:mm:ss' for legacy mode|
+|[#11419](https://github.com/NVIDIA/spark-rapids/issues/11419)|[FEA] Support Spark 3.5.3 release|
```

Comment:
Not sure how Spark 3.5.3 ended up here since the issue itself has no roadmap.

Reply:
the archive doc change was simply move content from previous changelog, so I suppose at that moment 24.12, these 2 items were marked as part of roadmap 24.12